### PR TITLE
add step 3 of setup

### PR DESCRIPTION
**Pull request by unmeshvrije**
They have to install mysql js drivers to use MySQL with node.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,13 @@ mysql> grant all privileges on *.* to 'hyfuser'@'localhost';
 mysql> create database userdb;
 ```
 
+**Step 3: Installing MySQL drivers for node**
+
+We want to use MySQL with JavaScript and to this end, we use `mysqljs` driver.
```
Comment (NoerGitKat) [inline README.md]: I'd add a hyperlink to the documentation: https://github.com/mysqljs/mysql.

File: README.md [hunk 2]
```diff
@@ -67,6 +67,13 @@ mysql> grant all privileges on *.* to 'hyfuser'@'localhost';
...
+* Use `node install mysql` command to install them.
```
Comment (NoerGitKat) [inline README.md]: It should be `npm install mysql`. I'd also consider asking students to install it globally, by adding the `-g` to it:

`npm install -g mysql`

File: README.md [hunk 3]
```diff
@@ -67,6 +67,13 @@ mysql> grant all privileges on *.* to 'hyfuser'@'localhost';
...
+* Run `node connection-test.js` from VScode (Windows/MAC) or the terminal (Linux/MAC).
```
Comment (NoerGitKat) [inline README.md]: We can be clearer here, by mentioning that the `connection-test.js` file can be found within the directory. Students can navigate there using the CLI and then execute the command.

File: README.md (hunk 1 above)
Comment (unmeshvrije) [reply README.md]: Added in the new commit.

File: README.md (hunk 2 above)
Comment (unmeshvrije) [reply README.md]: Thanks for spotting this. I have added the command

File: README.md (hunk 3 above)
Comment (unmeshvrije) [reply README.md]: Mentioned in the new commit

**Pull request merged**